enterprise_access/apps/api/v1/views.py

```diff
@@ -993,6 +1008,30 @@ def redeem(self, request, *args, **kwargs):
         Redeem a policy for given `learner_id` and `content_key`
 
         URL Location: POST /api/v1/policy/<policy_uuid>/redeem/?learner_id=<>&content_key=<>
+
+        status codes:
```

Comment:
inb4 somebody says "use drf-spectacular": my response is "that's a different ticket, probably [ENT-6948](https://2u-internal.atlassian.net/browse/ENT-6948)"